scriptHelper.js

```diff
@@ -3,39 +3,105 @@ require('isomorphic-fetch');
 
 function addDestinationInfo(document, name, diameter, star, distance, moons, imageUrl) {
    // Here is the HTML formatting for our mission target div.
-   /*
+    let planetDetails = `
                 <h2>Mission Destination</h2>
                 <ol>
-                    <li>Name: </li>
-                    <li>Diameter: </li>
+                    <li>Name: ${name}</li>
+                    <li>Diameter: ${diameter}</li>
                     <li>Star: ${star}</li>
-                    <li>Distance from Earth: </li>
-                    <li>Number of Moons: </li>
+                    <li>Distance from Earth: ${distance}</li>
+                    <li>Number of Moons: ${moons}</li>
                 </ol>
-                <img src="">
-   */
+                <img src = ${imageUrl}>
+    `;
+    document.getElementById ("missionTarget").innerHTML = planetDetails;
 }
 
 function validateInput(testInput) {
-   
+    let transformation = Number(testInput);
```

Comment:
This part isn't necessary. The isNaN check actually takes in a string. But this line converts your string into a number.